lib/kubepug_test.go

```diff
@@ -153,13 +152,14 @@ func TestGetDeprecated(t *testing.T) {
 		require.Nil(t, result)
 	})
 
+	configString := "/blabla123/kconfig"
 	t.Run("bad k8s config should fail", func(t *testing.T) {
 		pug := &Kubepug{
 			Config: &Config{
 				GeneratedStore: ts.URL + dataJSON,
 				K8sVersion:     "v1.22",
 				ConfigFlags: &genericclioptions.ConfigFlags{
-					KubeConfig: pointer.String("/blabla123/kconfig"),
+					KubeConfig: &configString,
```

Comment:
question: why change this? the tests are passing as is

Reply:
dependabot PR #495 and and #494 are failing because `k8s.io/utils/pointer` is deprecated. See lint failure [here](https://github.com/kubepug/kubepug/pull/495/files)
``` 
SA1019: "k8s.io/utils/pointer" is deprecated: Use functions in k8s.io/utils/ptr instead: ptr.To to obtain a pointer, ptr.Deref to dereference a pointer, ptr.Equal to compare dereferenced pointers.  (staticcheck)
```
As a result kubepug is unable to update cli-runtime or apimachinery until `pointer.String()` is replaced.

Reply:
ah ok, the fix was added here, https://github.com/kubepug/kubepug/pull/504, please rebase

Reply:
ok, I see you fixed dependabot yesterday as part of #504. I have deleted this commit.